src/txmempool.cpp

```diff
@@ -413,6 +413,7 @@ static CTxMemPool::Options&& Flatten(CTxMemPool::Options&& opts, bilingual_str&
 CTxMemPool::CTxMemPool(Options opts, bilingual_str& error)
     : m_opts{Flatten(std::move(opts), error)}
 {
+    m_txgraph = MakeTxGraph(64, 101'000, 10'000);
```

Comment:
n.b.

Want to see these magic values disappear 

91d9bfcca62bd6c168476f6af442bd7049ff872e